cirq-google/cirq_google/json_test_data/PhasedFSimCalibrationResult.json_inward

```diff
@@ -53,6 +53,7 @@
     "characterize_zeta": true,
     "characterize_chi": false,
     "characterize_gamma": false,
-    "characterize_phi": true
+    "characterize_phi": true,
+    "readout_error_tolerance": 0.4
```

Comment:
this is the `_inward` file - so maybe it would be useful to keep this without the `readout_error_tolerance` field to test the backward compatibility?

Reply:
I was a little bit confused with the `_inward` files. Reverted.

Reply:
You can copy the old version of the json files (without the readout_error_tolerance field) and name them with the inward suffix so it checks that old versions of the serialized data can still be read in.

Not super high priority for these cirq_google files under active development where we can intuit that it should load in just fine since the new field is optional.